admin/common_auth/views.py

```diff
@@ -24,3 +25,31 @@ def login(request):
 def logout(request):
     logout_user(request)
     return redirect('auth:login')
+
+def register(request):
+    # Creates User, takes selected permissions and assigns it to user, then saves resetform for email invitation
+    if request.user.is_authenticated():
+        return redirect('auth:login')
```

Comment:
Might want to put a flash message call here to tell the user why they can't register an account before redirecting them just for user-friendliness.